securedrop/alembic/versions/c5a02eb52f2d_dropped_session_nonce_from_journalist_.py

```diff
@@ -0,0 +1,41 @@
+"""dropped session_nonce from journalist table and revoked tokens table due to new session implementation
```

Comment:
This schema change needs a test, see https://docs.securedrop.org/en/latest/development/database_migrations.html#unit-testing-migrations

For the dropped table, you can try a plain `SELECT * FROM revoked_tokens` and verify it raises a SQL error (there should be some examples of that in other tests)